### PR TITLE
InnerHTMLParserFastpathStudy nightly,beta 50%

### PR DESCRIPTION
**Pull request by atuchin-m**
For https://github.com/brave/brave-variations/issues/652

### CODE REVIEW

**Review by iefremov — DISMISSED** [review]

**Comment by atuchin-m** [comment]
Rebased

**Review by iefremov — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. 👍

**Pull request merged**
